test/parallel/test-debug-signal-cluster.js

```diff
@@ -29,6 +29,10 @@ let pids;
 
 child.stderr.on('data', function(data) {
   const childStderrOutputString = data.toString();
+  if (/DeprecationWarning: node --debug is deprecated\./
+    .test(childStderrOutputString))
```

Comment:
This seems to rely on only one line coming in per data event whereas subsequent lines suggest that it is possible that more than one line will come in at a time.

Rather than this check, it might be better to add the deprecation text three times to the `expectedContent` array defined earlier in the test.